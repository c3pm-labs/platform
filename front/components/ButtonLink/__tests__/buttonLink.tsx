import { render } from '@testing-library/react';

import ButtonLink from '../index';

test('ButtonLink', () => {
  const { container } = render(
    <ButtonLink href="https://c3pm.io/">Toto</ButtonLink>,
  );

  expect(container).toMatchSnapshot();
});
