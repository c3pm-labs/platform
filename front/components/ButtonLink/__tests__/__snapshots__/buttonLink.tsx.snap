// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonLink 1`] = `
<div>
  <a
    aria-disabled="false"
    class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-default-1 makeStyles-primary-2 makeStyles-primary-6 MuiButton-disableElevation"
    href="https://c3pm.io/"
    tabindex="0"
  >
    <span
      class="MuiButton-label"
    >
      Toto
    </span>
    <span
      class="MuiTouchRipple-root"
    />
  </a>
</div>
`;
